bindings/nodejs/index.d.ts

```diff
@@ -32,36 +32,237 @@ export interface PresignedRequest {
 }
 export class Operator {
   constructor(scheme: string, options?: Record<string, string> | undefined | null)
-  /** Get current path's metadata **without cache** directly. */
+  /**
+   * Get current path's metadata **without cache** directly.
```

Comment:
How will this generated in our API docs?

Reply:
![image](https://user-images.githubusercontent.com/24221472/228723968-b927bfd2-3338-4188-975e-6bfa057a697e.png)
Looks well.

Reply:
Wow, so nice!